packages/plugin-less/src/index.ts

```diff
@@ -75,7 +75,11 @@ const getLessLoaderOptions = (
   ): LessLoaderOptions => {
     const getLessOptions = () => {
       if (defaults.lessOptions && userOptions.lessOptions) {
-        return deepmerge(defaults.lessOptions, userOptions.lessOptions);
+        return deepmerge(defaults.lessOptions, userOptions.lessOptions, {
+          isMergeableObject: (obj) =>
```

Comment:
Can we use the internal `isPlainObject` helper here?

The same as https://github.com/web-infra-dev/rsbuild/blob/650004cee1376f9264673e61a79ff947b964587a/packages/core/src/helpers/index.ts#L72

Reply:
yeah, of course.